tools/telemetry-generator/src/handlers/appInsightsExecutionTimeTestHandler.ts

```diff
@@ -15,11 +15,11 @@ module.exports = function handler(fileData, telemetryClient: TelemetryClient): v
 		const arithmeticMeanMetricName = `${fileData.suiteName}_${testData.benchmarkName}_arithmeticMean`;
 		try {
 			console.log(
-				`emitting metric ${arithmeticMeanMetricName} with value ${testData.stats.arithmeticMean}`,
+				`emitting metric ${arithmeticMeanMetricName} with value ${testData.customData["Heap Used Avg"]}`,
```

Comment:
This is a handler for execution time tests so this line and 22 need to use "period (ns/op)" (double check the actual string, that's just from memory so it might not be exactly correct).